static/js/app.js

```diff
@@ -13,21 +13,31 @@ document.addEventListener('DOMContentLoaded', function() {
 		recipientLabel.style.visibility = 'hidden';
 	};
 
-	// TODO: multi file upload
+	// Multi file upload
 	const fileSelector = document.getElementById('file-selector');
 	fileSelector.addEventListener('change', (event) => {
 		const fileList = event.target.files;
 
-		form.elements['filename'].value = fileList[0].name;
-		const reader = new FileReader();
-		reader.addEventListener('load', (event) => {
-			var arrayBuffer = event.target.result;
-			var fileData = new Uint8Array(arrayBuffer);
-			encryptFile(fileData).then(function(encrypted_file) {
-				form.elements['attachment'].value = encrypted_file;
+		for (var i=0; i < fileList.length; i++) {
+			current_file = fileList[i];
+			filename_field = form.elements['filename-' + i];
```

Comment:
This throws `Uncaught TypeError: filename_field is undefined` in browser console if 6 files are selected.

Reply:
fixed in #4